layers/+tools/verb/packages.el

```diff
@@ -0,0 +1,47 @@
+;;; packages.el --- Verb layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2012-2017 Sylvain Benner & Contributors
+;;
+;; Author: Federico Tedin <federicotedin@gmail.com>
+;; URL: https://github.com/federicotdn/verb
```

Comment:
The link should be pointing to spacemacs ;; URL: https://github.com/syl20bnr/spacemacs